tests/pytests/test_vecsim.py

```diff
@@ -151,6 +151,32 @@ def del_insert(env):
     res = [4, 'a', ['v', vecs[0]], 'b', ['v', vecs[1]], 'c', ['v', vecs[2]], 'd', ['v', vecs[3]]]
     env.expect('FT.SEARCH', 'idx', '*=>[KNN 4 @v $b]', 'PARAMS', '2', 'b', 'abcdefgh', 'RETURN', '1', 'v').equal(res)
 
+def testReuse(env):
```

Comment:
This test is related to #2705 bug. The bug effected VECTOR index mostly, so I think it deserve a test in `test_vecsim.py`. It wasn’t part of the bug fix as it effects earlier versions that does not support VSS, so it would be easier to cherry-pick.

Reply:
I still find the name of the test a bit vague. **Maybe** something like `testReuseChangeToUnfitting` etc.
You can also mention 2705 in a comment BTW.